### ISSUE TITLE
Meta objects do not get added to the event payload

### ISSUE THREAD

**Issue by pablocoberly** [adrianhall/winston-splunk-httplogger#3]
I add metadata to the winston log entries using a rewriter as documented here:

https://www.npmjs.com/package/winston#filters-and-rewriters

`winston-splunk-httplogger` checks the length of `meta` before adding it to the payload:

```
if (meta) {
        if (meta instanceof Error) {
            payload.message.meta = {
                error: meta.message,
                name: meta.name,
                stack: meta.stack
            };
        } else if (meta.length) {
            payload.message.meta = meta;
        }
    }
```

`meta.length` is undefined, so it never gets added to the message. 

Happy to submit a pull request to fix this, but I wanted to check with you to make sure this wasn't by design, i.e. expecting meta to be an array instead of an object.

**Comment by pablocoberly** [comment]
Sorry, just noticed you fixed this some months ago, but npm package hasn't been updated.

**Comment by adrianhall** [comment]
I'll get a new package out this week.  Been meaning to do it for a while but need to set up a test environment.

Adrian Hall
________________________________

**Comment by pablocoberly** [comment]
@adrianhall, Many thanks. Is there anything I can do to help with this?

**Comment by adrianhall** [comment]
Just released v1.2.0 to npm.  Let me know if there are any issues.
